src/hist/histcustommodes.cpp

```diff
@@ -230,7 +242,7 @@ void HistCustomModes::zachariasAmplitudes(double temperature, unsigned ntime, ge
             }
           }
           if (energy<1e-6) continue; // avoid gamma acoustic modes
-          double sigma = sqrt( (phys::BoseEinstein(energy,temperature)+0.5)/(energy*phys::amu_emass) )
+          double sigma = sqrt( (phys::BoseEinstein(energy,temperature)+0.0)/(energy*phys::amu_emass) )
```

Comment:
```suggestion
          double sigma = sqrt( (phys::BoseEinstein(energy,temperature)+0.5)/(energy*phys::amu_emass) )
```
We are not doing this change in this PR since it is not related to strain :smile: